v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Parallel Statement Execution
+summary: The parallel statement execution feature allows parallel execution of multiple independent SQL statements within a transaction.
+toc: false
+---
+
+CockroachDB supports parallel execution of independent INSERT, UPDATE, UPSERT, and DELETE statements within a single transaction. 
+
+To execute statements in parallel, use the `RETURNING NOTHING` clause for the independent SQL statements.
+
+<div id="toc"></div>
+
+## Independent SQL statements
+
+CockroachDB considers SQL statements to be independent if they operate on independent tables and their execution can be safely reordered without affecting their results. 
```

Comment:
This jumps into how we classify independent statements without really discussing why that's important. I think we should pull up a lot of the detail we have in the examples to here. For instance, we first need to discuss how `RETURNING NOTHING` statements will be run asynchronously and will return immediately to the client before returning any result. We then need to discuss how statements running asynchronously will be run at the same time as long as that won't impact correctness. Finally, we can discuss how statement independence allows us to infer whether running statements concurrently affects correctness. After that, we'll need to discuss statement synchronization and error handling (which you discuss below, but which I think get lost down there).